requirements.txt

```diff
@@ -1,6 +1,6 @@
 xmlrunner
 tensorflow<2.0.0
-scikit-learn
+scikit-learn<0.24.0
```

Comment:
what was the error?  it would be best to fix the issue so we can support scikit-learn 0.24.0 asap

Reply:
I think I may have fixed the issue with this PR:
https://github.com/interpretml/interpret-community/pull/362
so lower-bounding to 0.24.0 might no longer be necessary